client/src/TenderList.js

```diff
@@ -65,18 +70,34 @@ const TendersList = () => {
 						<th>Tender Created Date</th>
 						<th>Tender Announcement Date</th>
 						<th>Tender Project Deadline Date</th>
+						<th>Description</th>
 						<th>Tender Status</th>
+						<th>Bids Received</th>
 					</tr>
 				</thead>
 				<tbody>
 					{tenders.map((tender) => (
 						<tr key={tender.id}>
-							<td>{tender.id}</td>
-							<td>{tender.title}</td>
+							<td>
+								<button onClick={() => handleTenderClick(tender.id)}>
+									{tender.id}
+								</button>
+							</td>
+							<td>
+								<button onClick={() => handleTenderClick(tender.id)}>
+									{tender.title}
+								</button>
+							</td>
 							<td>{new Date(tender.creation_date).toLocaleDateString()}</td>
 							<td>{new Date(tender.announcement_date).toLocaleDateString()}</td>
 							<td>{new Date(tender.deadline).toLocaleDateString()}</td>
+							<td>
+								{expandedTenderId === tender.id && (
+									<p>{tender.description || "No description available"}</p>
+								)}
+							</td>
```

Comment:
I think the idea was that we would truncate the content of the description and expand it when a `show more` button is clicked.

Anyway, let's table this task and come back to it later. Right now we need to focus on the client library, the UI, and the flow of everything we already have